Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs

```diff
@@ -470,6 +470,24 @@ public void ClearFallbackInputStack()
 
         #endregion Fallback Input Handler Options
 
+        #region IMixedRealityController Utilities
+
+        /// <inheritdoc />
+        public IMixedRealityController TryGetController(IMixedRealityInputSource inputSource)
```

Comment:
Should this be `public bool TryGetController(IMixedRealityInputSource inputSource, out IMixedRealityController controller)` to match some of our other `TryGet` methods? I'm not sure if we've been consistent with the naming/behavior everywhere, but all the ones I just found on my search behave this way.

Reply:
Sure I can update it to be that instead :)